src/styles/popups/InspectTask.scss

```diff
@@ -90,8 +63,3 @@
     margin-left: auto;
   }
 }
-
-.bts {
-  font-size: medium;
-  width: 15%;
-}
```

Comment:
when you remove my styling, please add new styling. Now several buttons look off, like "Close" on create Team or "Submit" for comments in InspectTask